### ISSUE TITLE
Translation: Spanish (es-ES)

### ISSUE THREAD

**Issue by ygneo** [github/opensource.guide#516]
Disclaimer:

This issue is to add spanish translation (es-ES) locale of the guides. There's an [issue](https://github.com/github/opensource.guide/issues/383) and a [PR](https://github.com/github/opensource.guide/pull/434), but PR build fails, and activity seems stalled.

Repository: https://github.com/ygneo/opensource.guide
Preview: https://ygneo.github.io/opensource.guide
Locale: `es-ES`

On your fork:

- [x] Update `title`, `description`, and `locale` in `_config.yml`, and push the branch to the `gh-pages` branch of your fork. This should build the website and it should be visible at the preview url above.
- [x] Copy `_data/locale/en-US.yml` to a new file with the name of your locale and translate the strings
- [x] Create a new directory in `_articles/` for your locale and translate each guide:
  - [x] best-practices.md
  - [ ] building-community.md
  - [ ] code-of-conduct.md
  - [ ] finding-users.md
  - [ ] getting-paid.md
  - [ ] how-to-contribute.md
  - [ ] leadership-and-governance.md
  - [ ] legal.md
  - [ ] metrics.md
  - [ ] starting-a-project.md
- [ ] Update `CNAME` to include your locale (e.g. de.opensource.guide)

Once a translation is completed:

- [ ] Open a Pull Request on the [upstream repository](https://github.com/github/opensource.guide) to add your fork to `translations` in `_config.yml`

**Comment by MikeMcQuaid** [comment]
Closing in favour of https://github.com/github/opensource.guide/issues/383 which will be merged into this repository in some form. Would love your help on that.

**Comment by ygneo** [comment]
@MikeMcQuaid Sorry, but the issue #383 is for a different spanish locale (`es-AR`). The work in this issue is for the spanish locale `es-ES`. We have already translated several files in our fork, and I just updated the status on this issue.

**Comment by MikeMcQuaid** [comment]
@ygneo When the translation is complete (note the translation process has changed): please open a PR to add it to this repo. Thanks!

**Comment by ygneo** [comment]
Hi again @MikeMcQuaid 

We've seen that spanish translation is already deployed, and it is accessible through the language select box. That's the argentinian spanish locale (`es-AR`), and we are wondering something. 

Once our translation (spanish locale `es-ES`) is ready, how are you going to add it to the site? A new entry in the language select box will be added? Are you going to end up having "Spanish" and "Spanish Argentinian" or something like that?

Thanks!

**Comment by MikeMcQuaid** [comment]
> Once our translation (spanish locale es-ES) is ready, how are you going to add it to the site? A new entry in the language select box will be added? Are you going to end up having "Spanish" and "Spanish Argentinian" or something like that?

We haven't decided yet but: yes, mostly likely.
